rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -275,6 +275,36 @@
     return result;
   }
 
+  /**
+   * This endpoint force cancels the claim of an existing Task.
+   *
+   * @param taskId the Id of the requested Task.
+   * @return the unclaimed Task.
+   * @throws TaskNotFoundException if the requested Task does not exist.
+   * @throws InvalidStateException if the Task is already in an end state.
+   * @throws InvalidOwnerException if the Task is claimed by a different user.
```

Comment:
This should not happen while forcing, should it?

Reply:
It shouldn't but we can't get rid of this because the Java API for forceCancelClaim calls cancelClaim(true) which has the InvalidOwnerException in its signature...